frontend/src/app/modules/log-in/log-in-page/log-in-page.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="log-in-body">
+    <div class="logo-part">
+        <svg xmlns="http://www.w3.org/2000/svg" width="118" height="118" viewBox="0 0 118 118" fill="none">
+            <circle cx="59" cy="59" r="59" fill="#61AD00"/>
+        </svg>
+        <div class="logo">
+            LOGO
+        </div>
+    </div>
+    <div class="log-in-part">
+        <div class="form-container">
+            <H4 class="sign-in-label">Sign in</H4>
+            <button class="third-party github">
+                Sign in with GitHub
+            </button>
+            <button class="third-party google">
+                Sign in with Google
+            </button>
+            <H5 class="or-label">or</H5>
+        
+            <form class="logInForm" [formGroup]="logInForm">
+                <label class="form-label">Email</label>
+                <input class="form-input" placeholder="Enter your name" formControlName="email"/>
+                <div class="label-and-forget-password">
+                    <div class="form-label">Password</div>
+                    <a href="someUrl" class="forget-password-link">Forget password?</a>
+                </div>
+                <p class="password-cont">
+                    <input class="form-input password-input" type="{{ showPassword ? 'text' : 'password' }}" placeholder="&#9679;&#9679;&#9679;&#9679;&#9679;" formControlName="password"/>
+                    <fa-icon [icon]="eyeIcon" style="color: #91919380" (click)="toggleShow()"></fa-icon>
```

Comment:
please do not use inlune styles

Reply:
Done as you suggested

Reply:
still see inline styles

Reply:
inline styles is this =)
![image](https://github.com/BinaryStudioAcademy/bsa-2023-leetwars/assets/38536017/36f7d527-0534-4aca-bff3-d56ef1a0e366)